manifest/no_manifest_provider.go

```diff
@@ -0,0 +1,13 @@
+package manifest
+
+import (
+	"context"
+)
+
+var _ ManifestProvider = (*NoManifestProvider)(nil)
+
+type NoManifestProvider struct{}
```

Comment:
Subjective: 
* `Noop` instad of No.
* While at it rename the files containing provider implementations in this package to either have `_mainfest_provider.go` suffix, or not.  I would probably use `_provider.go`.

Reply:
Yeah, I like the suggestion.